vendir.json

```diff
@@ -0,0 +1,10 @@
+{
+    "description": "Carvel vendir is a tool that makes it easy to vendor portions of git repos, github releases, helm charts, docker image contents, etc. declaratively",
+    "notes": "See https://carvel.dev/vendir/#examples for some examples",
+    "version": "0.21.1",
+    "homepage": "https://carvel.dev/vendir/",
+    "license": "https://github.com/vmware-tanzu/carvel-vendir/blob/develop/LICENSE",
```

Comment:
```suggestion
    "license": {
      "identifier": "Apache-2.0",
      "url": "https://raw.githubusercontent.com/vmware-tanzu/carvel-vendir/release/LICENSE"
    },
```

Reply:
If the identifier is a common one, we don't need URL right?

Reply:
> If the identifier is a common one, we don't need URL right?

@rashil2000 Not in my opinion. It's just noise.